packages/renderless/src/color-select-panel/linear-gradient/index.ts

```
@@ -0,0 +1,143 @@
+      }
+    })
+  }
+  const getRelativePos = (points: IColorPoint) => {
+    const bar = vm.$refs[LINEAR_GRADIENT_BAR] as HTMLElement
+    if (!bar) {
+      return 0
+    }
+    const rect = bar.getBoundingClientRect()
+    return Number.parseInt(((points.cursorLeft / rect.width) * 100).toFixed(0))
+  }
+  const toString = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)